common/time/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
```

Comment:
?

Reply:
No reason to allow unsafe in there, no?

Reply:
interesting, I didn't realize we were doing this so much, I think we should investigate to transform this into a whitelist instead of a blacklist! (but later :D)